### DISCUSSION TITLE
Search term reset button: how to change button tooltip text or remove it

### DISCUSSION THREAD

**Discussion by salswa** [algolia/instantsearch#5619] [Ideas]
how to change reset button tooltip text "Clear the search query"or remove it

**Comment by dhayab** [comment]
Hi, it depends on the flavor/variant of InstantSearch you're using.

For InstantSearch.js, it is not easily possible currently. You'll need to create your own UI using the [search box connector](https://www.algolia.com/doc/api-reference/widgets/search-box/js/#connector).

For other flavors, you can specify your desired label by adding an attribute/prop. You can check out the relevant API reference for more detail.
